### PR TITLE
fix for first hub mission & more gear for Dr.Prado

### PR DESCRIPTION
**Pull request by casswedson**
#### Summary
None

#### Purpose of change
a while back I ran into this rare error involving sunflowers and dirt; get rid as the game suggest, add a flag to remove all that's there before placing our custom mapgen

also add a little more loot to Dr. Prado the woman killed by the hub prototype robot you take care of in the first mission add some paper, writing utensils and a laptop for the report she's probably making
a little water, sunglasses and gum - damage the spawned items, the murder bot presumably trashed the gear

the error message:
Screenshot from 2023-05-14 15-01-36

#### Testing
sample
Screenshot from 2023-06-25 13-57-10

### CODE REVIEW

**Comment by KHeket** [comment]
May be also add at least modular headgear with prototype helmet to the dead body drop? Because, if the prototype greaves and vambraces can be obtained via dialog with intercom, the prototype helmet in unabtainable, and it is very big pain to buy modular headgear, the intercome sometimes just dont have it for mounths. It will be also logical freindly - torso and head are most vital parts, rather than legs and arms

**Comment by John-Candlebury** [comment]
> May be also add at least modular headgear with prototype helmet to the dead body drop? Because, if the prototype greaves and vambraces can be obtained via dialog with intercom, the prototype helmet is unabtainable, and it is very big pain to buy modular headgear, the intercome sometimes just dont have it for mounths. It will be also logical freindly - torso and head are most vital parts, rather than legs and arms

Lack of headgear is intentional, it should be introduced with other questlines.

**Review by John-Candlebury — APPROVED** [review]

File: data/json/npcs/robofac/NC_ROBOFAC_SCIENTIST.json [hunk 1]
```diff
@@ -4,11 +4,18 @@
     "id": "NC_ROBOFAC_SCIENTIST_worn",
     "subtype": "collection",
     "entries": [
-      { "item": "under_armor" },
-      { "item": "under_armor_shorts" },
-      { "item": "robofac_jumpsuit" },
-      { "group": "robofac_mantle_prototype" },
-      { "item": "coat_lab" },
+      { "item": "under_armor", "damage": [ 0, 2 ] },
```
Comment (Maleclypse) [inline data/json/npcs/robofac/NC_ROBOFAC_SCIENTIST.json:7]:
```suggestion
      { "item": "under_armor", "damage": [ 0, 2 ] }
```

File: data/json/npcs/robofac/NC_ROBOFAC_SCIENTIST.json [hunk 2]
```diff
@@ -4,11 +4,18 @@
     "id": "NC_ROBOFAC_SCIENTIST_worn",
     "subtype": "collection",
     "entries": [
-      { "item": "under_armor" },
-      { "item": "under_armor_shorts" },
-      { "item": "robofac_jumpsuit" },
-      { "group": "robofac_mantle_prototype" },
-      { "item": "coat_lab" },
+      { "item": "under_armor", "damage": [ 0, 2 ] }
```
Comment (Maleclypse) [inline data/json/npcs/robofac/NC_ROBOFAC_SCIENTIST.json:7]:
```suggestion
      { "item": "under_armor", "damage": [ 0, 2 ] },
```

**Comment by Maleclypse** [comment]
Reset tests.

**Pull request merged**